src/v/redpanda/admin_server.h

```diff
@@ -139,7 +163,14 @@ class admin_server {
 
               log_request(req, auth_state);
 
-              return handler(req, reply);
+              /// Intercept and log exceptions
+              try {
+                  return handler(req, reply);
+              } catch (...) {
+                  log_exception(
```

Comment:
It looks like `register_route` was updated with future-ized exception handling, but `register_route_raw` is still only catching exceptions thrown synchronously

Reply:
Yes that's because the `handler` passed to `register_route_raw` is one that does not return a future:
```
typedef std::function<sstring(const_req req, reply&)> handle_function;
```